pages/docs/contributing/automated-tests.mdx

```diff
@@ -4,7 +4,7 @@ import DocsLayout from "../../../components/layouts/DocsLayout.js";
 
 We use [Playwright](http://playwright.dev) for writing automated end to end (e2e) tests.
 
-1. Install Playwright dependencies `npx playwright install --with-deps`
```

Comment:
This step is still required, we discussed adding the `npm ci` step before this existing step not replacing it

Reply:
okay, my bad. Just to understand better, ' npm ci` should be step 1 followed by other steps correct? and do we add the warning as well along with step 1?

Reply:
Yes please...

1. install dependencies with `npm ci`
2.Install Playwright dependencies `npx playwright install --with-deps`
3. Run tests `npm run test`

Reply:
Thank you @eddiejaoude